app/views/lists/_list.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= div_for(@lists) do |list| %>
```

Comment:
I think it's best practice to use partials with local variables instead of using an instance variable since it makes them more reusable in many cases. You could also use the `collection` option here: [3.4.5 Rendering Collections](http://guides.rubyonrails.org/layouts_and_rendering.html).